src/components/Details/Details.css

```diff
@@ -1,23 +1,109 @@
-.schedule {
-  margin-top: 25px;
+.container {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  width: 80%;
 }
 
-/* --------- MEDIA QUERIES -----------  */
+.image-container {
+  background-image: url('../../images/melissa-legette.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-color: #fff9e9;
+  overflow: hidden;
+  height: 20em;
+  width: 100%;
+  object-fit: cover;
+  overflow: hidden;
+  display: flex;
+  border-radius: 2em;
+}
 
-@media screen and (min-width: 620px) {
+.name-overlay {
+  position: absolute;
+  top: 25%;
+  width: 100%;
+  background-color: rgba(255, 255, 255, 0.6);
 }
 
-@media screen and (min-width: 900px) {
+.market-name {
+  text-align: center;
+  color: black;
+  padding: 0.5em;
+  font-size: 1.5em;
 }
 
-@media screen and (min-width: 1200px) {
+.previous-icon {
+  width: 3em;
+  height: 3em;
+  position: absolute;
+  top: 1em;
+  left: 1em;
+}
+
+.pin-icon {
+  height: 2em;
+  width: auto;
 }
 
 .market-details {
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  background-color: #14b37e;
+  background-color: #fff9e9;
+  width: 100%;
+  min-height: 70%;
+  padding: 1.5em;
+  padding-top: 31px;
+  border-radius: 2em 0 2em 2em;
+  text-align: center;
+  position: absolute;
+  top: 15em;
+}
+
+.market-details::before {
+  content: '';
+  position: absolute;
+  background-color: transparent;
+  box-shadow: 0 30px 0 0 #fff9e9;
+  top: -4em;
+  right: 0;
+  height: 4em;
+  width: 4em;
+  border-bottom-right-radius: 2em;
+}
+
+.location-details,
+.schedule {
+  border: 1px solid #ee922f;
+  margin-bottom: 1em;
+  padding: 1em;
+}
+
+.location-details p {
+  margin-bottom: 0.25em;
+}
+
+.list li {
+  margin-left: 2em;
+  text-align: left;
 }
 
+/* --------- MEDIA QUERIES -----------  */
+
+@media screen and (min-width: 620px) {
+  .market-name {
+    font-size: 1.75em;
+  }
+}
+
+@media screen and (min-width: 900px) {
+  .market-name {
+    font-size: 2em;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+}
```

Comment:
Is this changing anything right now?

Reply:
No, there are def extra media queries in every css file. I will remove as needed!